src/lib/cozy-client/slices/synchronization.js

```diff
@@ -36,14 +36,38 @@ const doctypeSync = (state = doctypeSyncInitialState, action) => {
   }
 }
 
-const synchronization = (state = {}, action) => {
+const syncInitialState = {
+  started: false,
+  initialStatus: 'pending',
```

Comment:
Does it make sense to call it `initialStatus ` and not `status`?

Reply:
yes, because the status of some doctypes could change with the subsequent syncs

Reply:
Aaaah, I see. Ok then!